playwright-tests/ai-page.spec.ts

```diff
@@ -1,10 +1,19 @@
 import { test, expect } from "@playwright/test"
+import { viewports } from "./viewports"
 
-test("/ai", async ({ page }) => {
-  await page.goto("http://127.0.0.1:5177")
-  await page.click("text='Fake testuser Login'")
-  await page.waitForSelector(".login-avatar")
-  await page.click(".header-button[href='/ai']")
-  await page.waitForSelector("text='Submit a prompt to get started!'")
-  await expect(page).toHaveScreenshot()
-})
+for (const [size, viewport] of Object.entries(viewports)) {
+  test(`AI Page on ${size} screen`, async ({ page }) => {
+    await page.setViewportSize(viewport)
+    await page.goto("http://127.0.0.1:5177/ai")
+
+    // it is only working for desktop beacuse fake testuser login is not appearing for mobile
+
+    // await page.goto("http://127.0.0.1:5177");
+    // await page.click("text='Fake testuser Login'");
+    // await page.waitForSelector(".login-avatar");
+    // await page.click(".header-button[href='/ai']");
+    // await page.waitForSelector("text='Submit a prompt to get started!'");
```

Comment:
Issue: Get ai page working on mobile

Reply:
@seveibar my PR is still under review #95 i'll change it to 4 cols after that i'll inform you.

Reply:
> Issue: Get ai page working on mobile

It taking snapshot perfectly but the problem is that fake tester login button only shows on lg devices in navbar so my can't take snapshot in mobile devices.